packages/grid/_modules_/grid/hooks/root/useApi.ts

```diff
@@ -23,7 +23,7 @@ export function useApi(apiRef: GridApiRef): void {
       if (event && isSynthenticEvent(event) && event.isPropagationStopped()) {
         return;
       }
-      apiRef.current.emit(name, params, event);
+      apiRef.current.emit(name, params, event, {});
```

Comment:
Why the empty object? Don't we want to pass the API to XGrid's users that are subscribing to the events? We could use the same `MuiCallbackOptions`.

Reply:
It can be added but after the latest comments, it should be `api` directly.

Reply:
Updated